client/src/questions-n-answers/sub-components/mini-components/userAnswer.jsx

```diff
@@ -145,44 +143,48 @@ class UserAnswer extends React.Component {
   }
 
   render() {
+    let uploadClass;
```

Comment:
not sure where this is being used, but you might be able to make this logic into a ternary statement or put this logic where the element is using it.